ocp_on_osp.yml

```diff
@@ -183,6 +183,43 @@
         dest: /home/stack/
         remote_src: yes
 
+    - name: Create ignition-configs with openshift-installer
+      shell: |
+        ./openshift-install --log-level=debug create ignition-configs --dir=/home/stack/{{ ocp_cluster_name }}/ > /home/stack/{{ ocp_cluster_name }}/create_ignition_configs.log 2>&1
+      args:
+        chdir: /home/stack/
+
+    - name: Backup the ignition-configs
+      copy:
+        src: "/home/stack/{{ ocp_cluster_name }}/{{ item }}.ign"
+        dest: "/home/stack/{{ ocp_cluster_name }}/{{ item }}.ign.orig"
+        remote_src: yes
+      with_items:
+        - master
+        - worker
+        - bootstrap
+
+    - name: Read original master.ign config file
+      slurp:
+        src: "/home/stack/{{ ocp_cluster_name }}/master.ign.orig"
+      register: master_ign
+      when: ocp_master_etcd_nvme
+
+    - name: Get master etcd on nvme ignition configuration
+      set_fact:
+        master_ign_config_base: "{{ master_ign['content'] | b64decode }}"
+        master_etcd_nvme_config: "{{ lookup('template', 'master-etcd.ign.json') }}"
```

Comment:
Can you please submit master-etcd.ign.json file?

Reply:
The file is [here](https://github.com/redhat-performance/jetpack/blob/13c1ac023fed5cf871c453789ad515ddf9814773/templates/master-etcd.ign.json).